# -*- coding: utf-8 -*-
# Generated by Django 1.9.2 on 2017-10-23 07:19
from __future__ import unicode_literals

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('dao', '0064_auto_20171023_1002'),
    ]

    operations = [
        migrations.CreateModel(
            name='ExceptionCase',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('case_context', models.TextField(verbose_name='\u5f02\u5e38\u5177\u4f53')),
                ('case_type', models.IntegerField(default=0, verbose_name='\u5f02\u5e38\u7c7b\u578b')),
                ('case_level', models.IntegerField(default=0, verbose_name='\u5f02\u5e38\u7a0b\u5ea6')),
                ('created_at', models.DateTimeField(auto_now_add=True, null=True, verbose_name='\u521b\u5efa\u65f6\u95f4')),
                ('case_user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL, verbose_name='\u5f02\u5e38\u7528\u6237')),
            ],
            options={
                'verbose_name_plural': '\u5f02\u5e38\u7ba1\u7406',
            },
        ),
    ]
